README.md

```diff
@@ -0,0 +1,20 @@
+# Lockbox Website #
+
+## Building Locally ##
+
+to build the GitHub-pages style locally, you can use the provided `Dockerfile` to create an image:
+
+```bash
+git clone https://github.com/mozilla-lockbox/mozilla-lockbox.github.io
+cd mozilla-lockbox.github.io
+docker build --tag mozilla-lockbox.github.io .
+```
+
+Then create and run a container to run a local Jekyll server:
+
+```bash
+cd mozilla-lockbox.github.io
+docker run --rm --tty --interactive --publish 4000:4000 --volume $PWD:/srv/jekyll mozilla-lockbox.github.io
```

Comment:
@linuxwolf Sorry if this is a "just on my machine" problem and I did something to my environment, but I've searched high and low and can't tell why I'm getting this error when I run this `docker run` command:

>su-exec: /usr/gem/bin/bundle: No such file or directory

I have no idea why _anything_ would expect `/usr/gem` to exist (it doesn't locally here). I feel like its related to this but can't tell: https://github.com/jekyll/docker/blob/master/repos/jekyll/Dockerfile

Reply:
\-:

Something is screwy with the base image, but apparently not for me.

Reply:
Looks like something with that base image broke between yesterday and today.  /sigh